go.mod

```diff
@@ -5,13 +5,13 @@ go 1.13
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.4.3+incompatible
 	github.com/hashicorp/vault => github.com/hashicorp/vault v1.2.0
-	github.com/mrparkers/terraform-provider-keycloak => github.com/pulumi/terraform-provider-keycloak v0.0.0-20200310160058-b554f515d900
+	github.com/mrparkers/terraform-provider-keycloak => github.com/pulumi/terraform-provider-keycloak v0.0.0-20200321130755-dfe55d7eb038
 )
 
 require (
 	github.com/hashicorp/terraform-plugin-sdk v1.7.0
 	github.com/mrparkers/terraform-provider-keycloak v0.0.0-00010101000000-000000000000
 	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi v1.9.1
-	github.com/pulumi/pulumi-terraform-bridge v1.6.4
+	github.com/pulumi/pulumi v1.12.1
```

Comment:
`1.12.1` has that issue with python docs generation that fails the release. Maybe start using `1.13.0` or the same master-based that I used for other providers.

Reply:
nice tip!